### DISCUSSION TITLE
Reset DB password, table editor now broken

### DISCUSSION THREAD

**Discussion by GizmoRay** [supabase/supabase#28929] [Questions]
![image](https://github.com/user-attachments/assets/4a39a10c-6a82-4c51-b1d7-53fc85114b75)

I don't want to restart server if possible, as it would impact production work stream.

**Comment by silentworks** [comment]
You should try restarting your project from the **General** settings page https://supabase.com/dashboard/project/_/settings/general. If this doesn't work then you should contact Supabase support https://supabase.com/dashboard/support/new
